### PR TITLE
 tests for signup & login

### PR DESCRIPTION
**Pull request by BehnazDer**

### CODE REVIEW

**Comment by BehnazDer** [comment]
The plain password is for testing so I think the security alert can be ignored. I will add more tests and dont want to merge this branch now

File: src/routes/__tests__/auth.routes.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  afterEach(() => {
+    return clearDatabase();
+  });
+  
+  afterAll(() => {
+    return closeDatabase();
+  });
+  
+  jest.mock('../../middleware/verifyToken', () =>
+    jest.fn((req, res, next) => {
```
Comment (irina-baeva) [inline src/routes/__tests__/auth.routes.test.js:24]: do u need verify token mock here?

File: src/controllers/auth.controllers.js [hunk 2]
```diff
@@ -15,7 +15,9 @@ export const signup = async (req, res, next) => {
     }
     const userExist = await User.findOne({ email: req.body.email });
     if (userExist) {
+      console.log(userExist)
       next(
```
Comment (irina-baeva) [inline src/controllers/auth.controllers.js:19]: console.logs

File: src/routes/__tests__/auth.routes.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        expect(user.email).toBeTruthy();
+        expect(user.password).toBeTruthy();
+    });
+
+    test('should login the signedup user', async () => {
+        const salt = await bcrypt.genSalt(10);
+        const hashedPw = await bcrypt.hash(user1.password, salt)
+        const user = await User.create({...user1, password: hashedPw});
+        console.log(user)
+        let res = await request(app)
```
Comment (irina-baeva) [inline src/routes/__tests__/auth.routes.test.js:52]: console.log
try to run yarn lint before to check that it is all linted

File: src/utils/database.js [hunk 4]
```diff
@@ -37,7 +37,7 @@ export const closeDatabase = async () => {
 
 export const clearDatabase = async () => {
   const { collections } = mongoose.connection;
-  Object.keys(collections).forEach(collection =>
-    mongoose.connection.collection(collection).deleteMany({}),
+  Object.keys(collections).forEach( async collection =>
```
Comment (thdvbr) [inline src/utils/database.js:40]: O good catch i forgot to add await when refactoring! I think we dont need async on this line since we have it on the top

Comment (thdvbr) [reply src/utils/database.js:40]: ah actually we should keep async here and delete async on the top!

**Pull request closed without merging**